spec/unit/associations_spec.rb

```diff
@@ -39,8 +39,10 @@
   end
 
   describe Site do
-    if Mongoid::Compatibility::Version.mongoid6_or_newer?
+    if Mongoid::Compatibility::Version.mongoid6?
```

Comment:
This should be `Mongoid::Compatibility::Version.mongoid6_or_older?` to match the code in `site.rb`, right?

Reply:
Yeah, there's a mismatch, but it should be  ```Mongoid::Compatibility::Version.mongoid6``` on both files